robocup_ws/src/Planner/src/goalkeeper_controller.py

```diff
@@ -7,6 +7,9 @@
 from src.BasicCommonActions.RotateToPointAction import RotateToPointAction"""
 from robot_control import Robot, Goal
 from game_interfaces.msg import PlayerCommand
+import numpy as np
+from BasicCommonActions.go_to_action import simple_go_to_action, go_to_fast
+from game_interfaces.msg import Position
 
 """
```

Comment:
It's fine to delete the code of this part. From my prospective, these code is useless any more